### PR TITLE
Create Issue Templates

### PR DESCRIPTION
**Pull request by PCTipsGR**
Since I saw what you said on #3730, about not preferring long issue templates, I created my own version of simple issue templates that might help a new user with submitting an issue and not confuse him.

I added as little as I could as so to keep it brief and not long per your instructions to the submitter of the issue mentioned before.

I have allowed edit from maintainers and so if you find something too long or something that would prevent users from submitting an issue, tell me to remove it and I will. I am open to any change.

_If you want, you can remove it too, no need to tell me_

### CODE REVIEW

**Comment by Jaex** [comment]
> (eg. Windows)

It is always Windows :P 

> What did you expect to happen?

Isn't answer to this always will be "bug to not happen".

> Describe alternatives (Optional)

I have no idea what is this section for.

For bug template having section for reproduce steps will be nice because most of the time people not writes how can I reproduce their issue.

**Comment by Jaex** [comment]
Also in ShareX version section we can ask them to specify which build you are using (Steam, Windows Store etc.).

**Comment by PCTipsGR** [comment]
Thanks for your review @L1Q @Jaex, I corrected these points. 

Exactly as you mentioned GitHub gives you the option to open a regular issue so I though that Other was not needed, which is why I didn't create a template for it.

**Comment by stuntguy3000** [comment]
At the risk of overengineering, what if we were to load the template into ShareX itself and add like a "report a bug" button which sends you to GitHub and autofills some information

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+name: Bug report
+about: Create a report to help us fix any issues ShareX might have
+labels: 
+
+---
+
+**ShareX version and OS**
+Mention your current ShareX version and the OS you are on (eg. Windows)
```
[the remark this answers is not in the document]
Comment (PCTipsGR) [reply .github/ISSUE_TEMPLATE/bug_report.md]: This got already fixed in a previous comment of mine

**Comment by PCTipsGR** [comment]
> At the risk of overengineering, what if we were to load the template into ShareX itself and add like a "report a bug" button which sends you to GitHub and autofills some information

Adding a bug report button which has the template and submits it to an issue seems a good idea, I had actually never considered this.

**Comment by stuntguy3000** [comment]
image

**Comment by Jaex** [comment]
@stuntguy3000 error windows of ShareX already have report bug button.

**Comment by stuntguy3000** [comment]
In my opinion that's not sufficient enough as many bugs may not throw errors, or if an error dialogue is dismissed quickly (or the program crashes) the user may not know how to proceed with reporting the error.

**Pull request closed without merging**